lib/ClimaLandSimulations/src/fluxnet/US-Ha1/US-Ha1_parameters.jl

```diff
@@ -0,0 +1,125 @@
+"""Site-specific model parameters for running CliMA LSM on the Harvard Forest
+fluxtower site."""
+
+# Data File download link
+data_link = "https://caltech.box.com/shared/static/xixaod6511cutz51ag81k1mtvy05hbol.csv"
+
+# Timezone (offset from UTC in hrs)
+time_offset = 5
+
+# Site latitude and longitude
+lat = FT(42.5378) # degree
```

Comment:
yeah I totally agree - there's a lot of optimization that we could do. (putting those files into a package is a first step), I'll do as much as I can but it can probably be done incrementally in many PRs. (I see this work as refactoring the hardcoded experiments/integrated/fluxnet we had - eventually it should be all nicely organized)

Reply:
sounds good! no need to make those changes in this PR then